OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1624,7 +1704,7 @@ public void TestUnsupportedVariableType()
             Assert.IsNull(featureVariableStringRandomType);
 
             var featureVariableStringJsonType = Optimizely.GetFeatureVariableString("unsupported_variabletype", "string_json_key", TestUserId);
-            Assert.AreEqual(featureVariableStringJsonType, "{\"myvalue\": \"jsonValue\"}");            
+            Assert.IsNull(featureVariableStringJsonType);            
```

Comment:
Why value is changed to `IsNULL`

Reply:
because the subtype of that variable is json and we are changing type from string to json upon initialization. so it will not match and will return null as type of variable will not match